### ISSUE TITLE
Resource Monitor causing CPU usage

### ISSUE THREAD

**Issue by MagicShoebox** [Njanderson/resmon#18]
I recently experienced the Resource Monitor extension causing higher than expected CPU usage on Windows 10. This showed up in Task Manager as usage by Service Host: Windows Management Instrumentation. I traced the activity to WMI queries being run by Resource Monitor. On my machine, the extra usage was somewhere around 3-5% per VSCode window.

Obviously the extension will consume some resources, but I was surprised by how much CPU activity it was causing. It's possible something about my machine was causing the WMI Queries to consume more resources than they should.

Just wanted to note this in case anyone else was experiencing anything similar.

**Comment by letmaik** [comment]
@Njanderson Maybe this issue goes away by updating the `systeminformation` package?

**Comment by Njanderson** [comment]
Yeah! This uses systeminformation, because it exposes all the functionality that I wanted the most simply. However, this systeminformation package just calls wmi queries then parses them inline! It's crazy! I actually did follow up with the author and asked about using NodeFFI to call into native Win32 queries of this information and marshal back and forth, so we aren't spinning up a process, parsing the output, and he said, yes in due time; this is an issue, but he hasn't budget on it yet.

Love to have a more native solution, actually. Leaving this open.

**Comment by letmaik** [comment]
@Njanderson I would forget native solutions as publishing them via VS Code is a major pain. I really wouldn't do it until they officially support this in a sane way.
What I would do instead is to increase the default refresh rate to 5s. I think that's good enough and would reduce CPU usage.
